src/app/portlets/content-types/main/content-types.component.ts

```diff
@@ -98,9 +98,6 @@ export class ContentTypesPortletComponent implements OnInit {
 
             this.actionHeaderOptions = {
                 primary: {
-                    command: ($event) => {
```

Comment:
why do you remove it?

Reply:
That was reaaaally old from when we select the content type to create inside the creation page, now we have the submenu.